defaults/manager_persistent_snapshots.yaml

```diff
@@ -1,18 +1,79 @@
 aws:
+  bucket: "manager-backup-tests-permanent-snapshots-us-east-1"
   snapshots:
+    5:
```

Comment:
the structure of snapshot is a bit weird.

I would suggest switching it to a list, that we can search
size as key, and tags as keys, are not very helpful in this case, can can be one of the attribute of each item.

nested dict, isn't very nice to do lookups on

Reply:
Can you show an example? I'm not sure what structure you suggest

Reply:
A list of dicts

Using size as the key doesn't fit that we'll all the cases you have here.

Reply:
right now, we can start using this way, and when adding more content, we could refactor this dictionary

Reply:
O.k. I do recommend changing this, since it adds lots of logic to any code walking over this list